kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -1186,136 +1186,17 @@ private void convert(Map<String, Boolean> attributes, Att att) {
-            null,// 09
-            null,// 0a
-            null,// 0b
-            null,// 0c
-            null,// 0d
-            null,// 0e
-            null,// 0f
-            null,// 10
-            null,// 11
-            null,// 12
-            null,// 13
-            null,// 14
-            null,// 15
-            null,// 16
-            null,// 17
-            null,// 18
-            null,// 19
-            null,// 1a
-            null,// 1b
-            null,// 1c
-            null,// 1d
-            null,// 1e
-            null,// 1f
-            "Spce",// 20
-            "Bang",// 21
-            "Quot",// 22
-            "Hash",// 23
-            "Dolr",// 24
-            "Perc",// 25
-            "And",// 26
-            "Apos",// 27
-            "LPar",// 28
-            "RPar",// 29
-            "Star",// 2a
-            "Plus",// 2b
-            "Comm",// 2c
-            "-",// 2d
-            "Stop",// 2e
-            "Slsh",// 2f
-            "0",// 30
-            "1",// 31
-            "2",// 32
-            "3",// 33
-            "4",// 34
-            "5",// 35
-            "6",// 36
-            "7",// 37
-            "8",// 38
-            "9",// 39
-            "Coln",// 3a
-            "SCln",// 3b
-            "-LT-",// 3c
-            "Eqls",// 3d
-            "-GT-",// 3e
-            "Ques",// 3f
-            "-AT-",// 40
-            "A",// 41
-            "B",// 42
-            "C",// 43
-            "D",// 44
-            "E",// 45
-            "F",// 46
-            "G",// 47
-            "H",// 48
-            "I",// 49
-            "J",// 4a
-            "K",// 4b
-            "L",// 4c
-            "M",// 4d
-            "N",// 4e
-            "O",// 4f
-            "P",// 50
-            "Q",// 51
-            "R",// 52
-            "S",// 53
-            "T",// 54
-            "U",// 55
-            "V",// 56
-            "W",// 57
-            "X",// 58
-            "Y",// 59
-            "Z",// 5a
-            "LSqB",// 5b
-            "Bash",// 5c
-            "RSqB",// 5d
-            "Xor-",// 5e
-            "Unds",// 5f
-            "BQuo",// 60
-            "a",// 61
-            "b",// 62
-            "c",// 63
-            "d",// 64
-            "e",// 65
-            "f",// 66
-            "g",// 67
-            "h",// 68
-            "i",// 69
-            "j",// 6a
-            "k",// 6b
-            "l",// 6c
-            "m",// 6d
-            "n",// 6e
-            "o",// 6f
-            "p",// 70
-            "q",// 71
-            "r",// 72
-            "s",// 73
-            "t",// 74
-            "u",// 75
-            "v",// 76
-            "w",// 77
-            "x",// 78
-            "y",// 79
-            "z",// 7a
-            "LBra",// 7b
-            "Pipe",// 7c
-            "RBra",// 7d
-            "Tild",// 7e
-            null// 7f
-    };
+    private static String[] asciiReadableEncodingKoreCalc() {
+        String[] koreEncoder = StringUtil.asciiReadableEncodingDefault;
```

Comment:
are you sure this is actually doing a copy? it could just be assigning a pointer to the object, which would mean you're overwriting the default. You might want to use Arrays.copyOf.

Reply:
Done.